docs/backtracking/n-queens.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: N Queens
+slug: /backtracking/n-queens
+---
+
+## Introduction
+
+In this problem, we are provided an N x N chessboard, and we have to find if there exists any way to place N chess queens such that no two queens can attack each other. The possible directions where the queen can move and attack are horizontal, vertical, and diagonal. If there exists any way then the output is going to be the corresponding positions of N queens else `false` will be returned.
+
+## Explaination
+
+This problem can be solved using `backtracking`. Each row and column will contain only one queen. We will start from the first cell of the chessboard. We will move row-wise, and for each row, we will see if there exists a column where the queen can be placed safely, i.e., no two queens are at an attacking position, we will include that cell in solution and will proceed further, and if we didn't find any such cell, then we will return false and backtrack to previous cell, change the queen's position.
```

Comment:
`We will move row-wise, and for each row, we will see if there exists a column where the queen can be placed safely, i.e., no two queens are at an attacking position, we will include that cell in solution and will proceed further, and if we didn't find any such cell, then we will return false and backtrack to previous cell, change the queen's position.`

Break this into smaller lines.